test/integration/harvest/IdleHarvestStrategy_USDT.js

```diff
@@ -116,24 +117,33 @@ describe.only("Idle Harvest Strategy (USDT)", async () => {
 
   it("Deposit", async () => {
     console.log("Idle Harvest Strategy", IdleHarvestStrategy.address);
+    let balanceBefore = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
     await USDT.connect(user).approve(IdleHarvestStrategy.address, AMOUNT_TO_TRANSFER);
     await IdleHarvestStrategy.connect(user).deposit(AMOUNT_TO_TRANSFER);
+    let balanceAfter = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
+    expect(balanceAfter).gt(balanceBefore);
   });
 
   it("Redeem", async () => {
     let AMOUNT_TO_REDEEM = BN("99000000");
     await USDT.connect(user).approve(IdleHarvestStrategy.address, AMOUNT_TO_TRANSFER);
     await IdleHarvestStrategy.connect(user).deposit(AMOUNT_TO_TRANSFER);
 
+    let balanceBefore = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
     await IdleHarvestStrategy.connect(user).redeem(AMOUNT_TO_REDEEM);
+    let balanceAfter = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
+    expect(balanceBefore).gt(balanceAfter);
   });
 
   it("Redeem Underlying", async () => {
     let toRedeem = BN("90000000");
     await USDT.connect(user).approve(IdleHarvestStrategy.address, AMOUNT_TO_TRANSFER);
     await IdleHarvestStrategy.connect(user).deposit(AMOUNT_TO_TRANSFER);
 
+    let balanceBefore = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
     await IdleHarvestStrategy.connect(user).redeemUnderlying(toRedeem);
+    let balanceAfter = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
+    expect(balanceBefore).gt(balanceAfter);
   });
 
   it("Price, apr and rewardTokens", async () => {
```

Comment:
can you add some assertions here too? eg initially the apr should be 0 and after the first interaction with the strategy it should be > 0